test/external_test_llvm.py

```diff
@@ -49,12 +49,12 @@ def test_add_sum_add(self):
 if __name__ == "__main__":
   unittest.main()
 
-  """
```

Comment:
Why is this bad, aren't quote strings fine? Tbh, these should probably be deleted.

Reply:
the linter considers it a string statement in this case, if it was under a function it would be fine